### PR TITLE
Plasma Stage 2 Review

### PR DESCRIPTION
**Pull request by Dan-Nolan**

### CODE REVIEW

File: projects/Plasma MVP/JavaScript/Create Plasma Block/completionMessage.md [hunk 1]
```diff
@@ -1 +1,5 @@
-You created a Plasma Block. Next we will create a Plasma Deposit.
\ No newline at end of file
+## Nice One!
+
+You created a Plasma Block. Next we will create a Plasma Deposit.
+
+![Pong](https://res.cloudinary.com/divzjiip8/image/upload/v1551847295/mnYeQ8B_y0lpdn.gif)
```
Comment (Roots2D) [inline projects/Plasma MVP/JavaScript/Create Plasma Block/completionMessage.md]: I don't know what GIF's you're added but I'm excited to see them in action!

File: projects/Plasma MVP/JavaScript/Create Plasma Block/details.md [hunk 2]
```diff
@@ -1 +1,25 @@
...
+A B C D E F G H 
+```
+
+> In this Merkle Tree, consider each joining of the letters a hashing of the two nodes. So `AB` `==` `Hash(A,B)`
+
+While all 8 of these transactions occurred within the Plasma Chain, all we need store in the Root Chain is one hash: `ABCDEFGH`. 
+
+As a client in this system, you can watch this block mined with proof that transaction `D` has been stored in the Root Chain and therefore your transfer is complete!
+
+Meanwhile, our plasma chain effectively batched 8 transactions into one block. Not too shabby!
```
Comment (Roots2D) [inline projects/Plasma MVP/JavaScript/Create Plasma Block/details.md]:
```suggestion
Meanwhile, our plasma chain effectively batched 8 transactions into one block. Not too shabby! 

For more on Merkle Trees, check out our [Merkle Tree Lesson](https://www.chainshot.com/lessons/5c36bf15143eed0017f579755ba1558bd9f997cd2e33eba7/stage/5c36bf15143eed0017f579755ba155e6d9f997cd2e33eba8).
```

File: projects/Plasma MVP/JavaScript/Create Plasma Block/completionMessage.md (hunk 1 above)
Comment (Dan-Nolan) [reply projects/Plasma MVP/JavaScript/Create Plasma Block/completionMessage.md]: 😆

Comment (Dan-Nolan) [reply projects/Plasma MVP/JavaScript/Create Plasma Block/completionMessage.md]: @K-Ho gave me the idea, he has some awesome GIFs in his chapters

File: projects/Plasma MVP/JavaScript/Create Plasma Block/details.md (hunk 2 above)
Comment (Dan-Nolan) [reply projects/Plasma MVP/JavaScript/Create Plasma Block/details.md]: great idea

**Pull request merged**